packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -16,7 +18,7 @@ const StyledSVG = styled.svg`
   background-color: ${props => props.theme.global.colors['light-4']};
   touch-action: pinch-zoom;
   max-width: ${props => props.$maxWidth};
-  ${props => props.$maxHeight && css`max-height: ${props.$maxHeight};`}
+  max-height: ${props => props.$maxHeight};
```

Comment:
Doesn't change anything in the UI, just removes the need for the `css` function.